lib/internal/modules/esm/loader.js

```diff
@@ -411,7 +411,8 @@ let emittedExperimentalWarning = false;
  * @returns {DefaultModuleLoader | CustomizedModuleLoader}
  */
 function createModuleLoader(useCustomLoadersIfPresent = true) {
-  if (useCustomLoadersIfPresent) {
+  if (useCustomLoadersIfPresent &&
+      !require('internal/modules/esm/utils').isLoaderWorker()) {
```

Comment:
I am not actually sure if in the loader worker we should just use the default loader for dynamic import and translators or wether loaders that are already loaded should affect subsequent imports. But then no tests complained when I just switched to the default loader for this 🤷‍♀️

Reply:
> wether loaders that are already loaded should affect subsequent imports in other loaders

They should, support for it was added in https://github.com/nodejs/node/pull/43772.

Reply:
We should already have tests for loaders affecting subsequent loaders, so if all the tests pass then this doesn’t break that behavior.

Reply:
Confirmed, it’s here: https://github.com/nodejs/node/blob/7dd3732480a84e1fa4fb194cd441dd59171fd6d4/test/es-module/test-esm-loader-chaining.mjs#L255